cryptoki-sys/oasis-headers/pkcs11.h

```diff
@@ -261,4 +264,7 @@ struct CK_FUNCTION_LIST {
 }
 #endif
 
+// Restore prior pack setting
+#pragma pack(pop)
+
```

Comment:
I believe explicit struct packing is really only needed for MSC based compilers.

Reply:
Not sure what you mean by this exactly. Are you saying that elsewhere, the default packing is what PKCS11 requires?

If so, then I'm not sure what to make of the bindings I generated. I was curious enough about that detail that I split the bindings updates into before- and after-pack commits to create a nice diff.